json_schema/1.0.0.schema.json

```diff
@@ -162,6 +169,7 @@
           }
         },
         {
+          "additionalProperties": false,
```

Comment:
I suppose it is still intended to not have `directory` and `versionJson`, if this object is matched. If so, I think that this won’t work because both fields are defined as properties in the object containing the `oneOf` field. I have not tested it yet though.

Reply:
Yes, that's the reason it is there. I have tested it and it does work, at least for me.